miteD/service/service.py

```diff
@@ -107,14 +132,72 @@ def _log_access(self, request, status, length):
             def _get_payload(msg):
                 return loads(msg.data.decode())
 
+            async def _send_notification(self, channel, msg):
+                await self._nc.publish(channel, dumps(msg).encode())
+
+            def _add_notify(self, cls):
+                if not self._notification_topics:
+                    return
+
+                notify = Notify()
+                channel_stem = 'notification.{}.{}'.format(self._layer, self._name)
+                for version in self._versions:
+                    for topic in self._notification_topics:
+                        channel = '{}.{}.{}'.format(channel_stem, version, topic)
+                        self._logger.info('Reigstering notifications channel: {}'.format(channel))
+                        setattr(notify, topic, partial(self._send_notification, channel))
```

Comment:
If I am reading this correct the partial here is to take care of the late-binding happening on `channel` yes?

Reply:
Also, typo on line 147. `Reigstering`

Reply:
That's some neat stuff dudes. I haven't worked with `partial()` much before. Cool way to set `notify`